### ISSUE TITLE
micro OS running on GBA

### ISSUE THREAD

**Issue by avivace** [gbdev/awesome-gbadev#1]
@Murplugg suggested in https://github.com/gbdev/awesome-gbdev/issues/148:

> Btw, there are more of these micro OS-es that runs on GBC / GBA, specifically:
> Nut/OS (GBA): www.ethernut.de/api/group__xg_nut_arch_arm_gba.html

> gbaunix: http://www.kernelthread.com/publications/gbaunix/, https://github.com/JALsnipe/gbaunix-0.0

**Comment by Murplugg** [comment]
Right. I was late in updating that other post, was looking for more specific info to make the post more useful and forgot to re-post.
- Nut/OS: Also has a page on Wikipedia: https://en.wikipedia.org/wiki/Ethernut
- uCLinux: I mis-wrote as "Unix". It seems to only have been emulated so I'm not sure that's interesting after all.
